src/components/DpTreeList/DpTreeListNode.vue

```diff
@@ -223,16 +223,20 @@ export default {
       return str.substring(1, str.length)
     },
 
+    hasChildren () {
+      return this.children.length > 0
+    },
+
     hasDraggableChildren () {
       return this.isBranch && (this.options.dragLeaves || this.options.dragBranches) && this.draggable
     },
 
     hasToggle () {
-      return this.isBranch && (this.children.length > 0 || this.hasDraggableChildren)
```

Comment:
I think I rember that I added the `hasDraggableChildren` Flag to ensure that empty branches can be opended if the list is draggable but the branch is empty to allow dropping elements there. I am not sure if there something changed in the meanwhile, but we have to ensure, that we still can drop elements into empty branches

Reply:
I understand what you mean and have tested it: we can drop elements into the closed branch as well, it does not need to be opened for this functionality. I am also not sure if we need this part: [[commit link]](https://github.com/demos-europe/demosplan-ui/pull/1194/commits/3d6e5fc83811a79df0e4998447c80197565a790f), as it works without it and an extra spacer is not needed there.

tested it: added an element from another branch to an empty, closed branch → Success (the branch can now be expanded)